controllers/rhmi/rhmi_controller.go

```diff
@@ -1084,6 +1088,21 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 		}
 	}
 
+	log.Info("preflightChecks: checking if STS mode")
```

Comment:
Is there a reason why this is not part of our preflightChecks function instead? https://github.com/integr8ly/integreatly-operator/blob/f83a81dc2b6913c64c87d5163a7b2764f4da909d/controllers/rhmi/rhmi_controller.go#L946

Reply:
I can move that log line in the `preflightChecks` function. I don't think it will be a problem and makes sense

Reply:
Seems like it already is part of that function